include/quic/opt.hpp

```diff
@@ -13,18 +13,28 @@ namespace oxen::quic
         {
             using Address::Address;
         };
+
         struct remote_addr : public Address
         {
             using Address::Address;
         };
+
+        struct max_streams
+        {
+            int stream_count{};
```

Comment:
Maybe default this value to the constexpr constant?  That way passing a default-constructed `max_streams` would be the same as not passing it at all.